SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -247,9 +247,11 @@ SCDocHTMLRenderer {
 		<< "</script>\n"
 		<< "<script src='" << baseDir << "/scdoc.js' type='text/javascript'></script>\n"
 		<< "<script src='" << baseDir << "/docmap.js' type='text/javascript'></script>\n" // FIXME: remove?
-		<< "<script src='" << baseDir << "/frontend.js' type='text/javascript'></script>\n"
+		// << "<script src='" << baseDir << "/frontend.js' type='text/javascript'></script>\n" //does not exist
 		// QWebChannel access
-		<< "<script src='qrc:///qtwebchannel/qwebchannel.js' type='text/javascript'></script>\n"
+		// << "<script src='qrc:///qtwebchannel/qwebchannel.js' type='text/javascript'></script>\n" // does not exist
+		<< "<script src='https://polyfill.io/v3/polyfill.min.js?features=es6'></script>\n"
+		<< "<script id='MathJax-script' async src='https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js'></script>\n"
```

Comment:
Not a good idea -- this makes SCDoc connect to the Internet.

Reply:
Thank you very much! I added files to be able to use LaTeX and MathML offline.
[f129943](https://github.com/supercollider/supercollider/pull/6254/commits/f129943568073ad75baa90c0bb9a888c051e5588)

and changed the code:
https://github.com/supercollider/supercollider/pull/6254/commits/288a1c80bd9ee09793c9ffa5969c17dadddb858c#diff-16b57c89cf950f36adc60ebaf32b5fe84d3b6944a9132f91da1960c65e41a6b6R279-R287

https://github.com/supercollider/supercollider/blob/288a1c80bd9ee09793c9ffa5969c17dadddb858c/SCClassLibrary/SCDoc/SCDocRenderer.sc#L280-L286